common/build.gradle

```diff
@@ -13,7 +13,8 @@ android {
 }
 
 dependencies {
+    api "android.arch.lifecycle:runtime:$architectureVersion"
```

Comment:
This was `implementation` but is now two `api` dependencies.  Were we wrong before?

Reply:
No, I just moved this up the tree: we used to have it as `api` in both the `firestore` and `database` modules so I just pulled it up to the `common` module. Do you think we should leave it as it was in the lower modules?

Reply:
Ohhh I see.  SGTM!